ui/app/AppLayouts/Wallet/popups/AddAccountModal.qml

```diff
@@ -235,13 +249,17 @@ StatusModal {
                     advancedSelection.isValid
             }
 
+            highlighted: focus
+
             MessageDialog {
                 id: accountError
                 title: "Adding the account failed"
                 icon: StandardIcon.Critical
                 standardButtons: StandardButton.Ok
             }
 
+            Keys.onReturnPressed: nextButton.clicked(null)
```

Comment:
@Khushboo-dev-cpp for me it looks more readable, always, if you add the code to a function (neglect naming), like:
```
function nextAction() {
    //...
}
onClicked: root.nextAction()
Keys.onReturnPressed: root.nextAction()
```

Reply:
Done!